tests/integration/test_vectordraw.py

```diff
@@ -21,6 +22,10 @@ class TestVectorDraw(StudioEditableBaseTest):
     Test student view of VectorDrawXBlock.
     """
 
+    def setUp(self):
+        super(TestVectorDraw, self).setUp()
+        self.driver.implicitly_wait(3)
```

Comment:
This is the part that I *think* fixed the other flaky tests for me. This is supposed to tell selenium to poll the dom for up to 3 seconds if any element it's looking for isn't immediately available. A better method could be to find everywhere where an element is selected and add a 'wait for' along with a timeout, but this seems neat and achieves the same thing. :crossed_fingers:  

ref. [docs](https://selenium-python.readthedocs.io/waits.html#implicit-waits)

Reply:
Update: it did not fix the flaky tests on [travis](https://travis-ci.org/open-craft/xblock-vectordraw/jobs/649813354?utm_medium=notification&utm_source=github_status). :(

Reply:
Let's increase the timeout and hope for the best.

Reply:
Decreased the timeout because this wasn't related to the flaky tests. Left it there as a precaution though.